packages/salesforcedx-utils-vscode/src/commands/preconditionCheckers.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+import { workspace } from 'vscode';
+import { nls } from '../messages';
+import { Predicate, PredicateResponse } from '../predicates';
+import { PreconditionChecker, SFDX_PROJECT_FILE } from '../types';
+import {
```

Comment:
We should export these functions so we can import with just having `../workspaces`

Reply:
Updated, but since workspace is in a different folder, we'd still have multiple copies of workspaces code